README.md

```
@@ -1,110 +1,230 @@
-- **CSV.GZ/TSV.GZ**: Functionality is the same as CSV/TSV files
+All of the input formats are supported in .gz version as well.
+
+**Multiple sequence columns:** `evaluate-classes` analyzes each column separately, then concatenates all sequences for combined analysis. `evaluate-splits` only analyzes concatenated sequences.
+
+## Output Files
+
+Each command writes into its own sub-directory of `--out-folder` — `class/` for
+`evaluate-classes` and `split/` for `evaluate-splits` — then into one directory
+per sequence column, then into one directory per comparison:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)